bundles/org.openhab.core.io.bin2json/.classpath

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file seems to be missing entries for the Java source folders - at least they are not present, if I import the project in Eclipse.

Reply:
done